instrumentation/grpc-1.6/library/src/main/java/io/opentelemetry/instrumentation/grpc/v1_6/TracingClientInterceptor.java

```diff
@@ -43,6 +43,7 @@
   public <REQUEST, RESPONSE> ClientCall<REQUEST, RESPONSE> interceptCall(
       MethodDescriptor<REQUEST, RESPONSE> method, CallOptions callOptions, Channel next) {
     GrpcRequest request = new GrpcRequest(method, null, null);
+    request.setAuthority(next.authority());
```

Comment:
The Javadocs for `authority()` say the following:
```
The authority of the destination this channel connects to. Typically this is in the format host:port.
```
Is there any significant difference between capturing `authority()` and the `net.peer.*` attributes?